src/DCS-BIOS/ControlLocator/DCSBIOSControlLocator.cs

```diff
@@ -165,7 +165,6 @@ public static List<DCSBIOSControl> GetModuleControlsFromJson(string filename, bo
                             _dcsbiosControls.AddRange(controls);
                         }
                         result.AddRange(controls);
-                        PrintDuplicateControlIdentifiers(controls);
                     }
 
                     DCSBIOSAircraftLoadStatus.SetLoaded(filename, true);
```

Comment:
I find it a little disturbing that filename in input parameter is a 'real' filename 
`GetModuleControlsFromJson(string filename,..` 
then in the `SetLoaded()` function, the first parameter is named `profile`  
`public static void SetLoaded(string profile, bool loaded)`

Reply:
Yup, I have now renamed it to ```SetLoaded(string filename, bool loaded)```. Also removed all magic mentions of the meta modules. DCSAircraft now has contants for the file names.